core/services/workflows/v2/engine_test.go

```diff
@@ -53,3 +43,44 @@ func TestEngine_Init(t *testing.T) {
 	module.EXPECT().Close().Once()
 	require.NoError(t, engine.Close())
 }
+
+func TestEngine_Start_RateLimited(t *testing.T) {
+	t.Parallel()
+	sLimiter, err := syncerlimiter.NewWorkflowLimits(logger.TestLogger(t), syncerlimiter.Config{
```

Comment:
have t.Run("engine is globally rate limited", func(...) {}), missing `t.Run("engine is rate limited for owner", ...)`